packages/ai-chat-components/src/components/extended-button/src/extended-button.scss

```diff
@@ -10,6 +10,6 @@ $css--plex: true !default;
 @use "../../../globals/scss/vars" as *;
 @use "@carbon/web-components/scss/components/button/button";
 
-:host(#{prefix}-extended-button) {
```

Comment:
we can just delete the whole example "extended-button" now that we have a real component in here